Add capability for TOMAS simulations in GCHP

msulprizio opened the pull request:
### Name and Institution (Required)

Name: Melissa Sulprizio
Institution: Harvard / GCST

### Describe the update

This pull request adds the option to run GCHP with TOMAS as originally implemented by @BettyCroft in:

- https://github.com/geoschem/GCHP/pull/252

This pull request should be merged alongside:
- https://github.com/geoschem/geos-chem/pull/2060
- https://github.com/geoschem/HEMCO/pull/251

BettyCroft committed:
Updates for TOMAS in GCHP

BettyCroft committed:
TOMAS updates for HEMCO

BettyCroft committed:
Updates for CMakeLists.txt for TOMAS

BettyCroft committed:
Updates for TOMASS

BettyCroft committed:
Updates for TOMAS diagnostics and for GCHP

BettyCroft committed:
Updates for TOMAS

Commit:
resolved

Jourdan-He committed:
merge origin/dev

msulprizio committed:
Merge branch 'feature/TOMAS' into dev/14.1.0

Signed-off-by: Melissa Sulprizio <mpayer@seas.harvard.edu>

msulprizio committed:
Add further GEOS-Chem and HEMCO submodule updates for TOMAS in GCHP

Signed-off-by: Melissa Sulprizio <mpayer@seas.harvard.edu>

msulprizio committed:
Merge PR #252 (branch 'feature/TOMAS') into dev/14.2.0

Resolved conflicts in:
	CHANGELOG.md
	src/GCHP_GridComp/GEOSChem_GridComp/CMakeLists.txt
	src/GCHP_GridComp/GEOSChem_GridComp/geos-chem
	src/GCHP_GridComp/HEMCO_GridComp/HEMCO

Signed-off-by: Melissa Sulprizio <mpayer@seas.harvard.edu>

msulprizio committed:
GEOS-Chem submodule update: Add fixes following merge of TOMAS updates

Signed-off-by: Melissa Sulprizio <mpayer@seas.harvard.edu>

msulprizio committed:
Merge branch 'main' into dev/tomas

Signed-off-by: Melissa Sulprizio <mpayer@seas.harvard.edu>

yantosca commented on CHANGELOG.md:
```diff
@@ -36,6 +40,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Fixed
 - Fixed post-advection pressure edges (PLE1) passed to advection to be derived from the correct surface pressure
 - Fixed typo in `docs/source/conf.py`, "_static" should be "_static/"
+>>>>>>> origin/main
```
Delete leftover slug from the git merge

msulprizio replied:
This is now fixed in https://github.com/geoschem/GCHP/pull/359/commits/41c136fc1e725e98a54216d7df2bcdb94f7fbaa3.

yantosca reviewed (changes requested):
This looks mostly good to merge, pending minor fixes as noted.

Also: The HEMCO submodule only has 2 files changed but the HEMCO PR #251 contains changes 3 files.  We'll need to point to the updated commit in the HEMCO submodule.

msulprizio committed:
Fix CHANGELOG.md to remove line leftover from merge

Also bring dev/tomas branches in the GEOS-Chem and HEMCO submodules
up-to-date.

Signed-off-by: Melissa Sulprizio <mpayer@seas.harvard.edu>

yantosca reviewed (approved):
Now good to merge!
